src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -160,7 +146,15 @@ function validateChildKeys(node, parentType) {
         var step;
         while (!(step = iterator.next()).done) {
           if (ReactElement.isValidElement(step.value)) {
-            validateExplicitKey(step.value, parentType);
+            const getMainExplicitKeyErrorMessage = getExplicitKeyErrorMessage.bind(
```

Comment:
Same here. If we bind outside of the loop (maybe at the top of `validateChildKeys`) we can save some work